training/datastore/s3_connection.py

```diff
@@ -66,11 +75,11 @@ def load_raw_data(self):
 
     def load_existing_data(self):
         """Load the node registry dump from S3 bucket."""
-        NPM_clean_json_data_path = os.path.join("npm", "dev", "2019-01-03",
+        NPM_clean_json_data_path = os.path.join("2019-01-03",
                                                 "data/node-package-details-with-url.json")
-        if (self.s3_client.object_exists(NPM_clean_json_data_path)):
+        if self.s3_client.object_exists(NPM_clean_json_data_path):
             try:
-                logger.info("Path Existed")
+                logger.info("Reading dump data from 2019-01-03 version.")
```

Comment:
This shouldn't be hardcoded should it?

Reply:
It should be because the data dump exists only in the 2019-01-03 version :).